articles/connections/passwordless/sms-gateway.md

```diff
@@ -1,66 +1,74 @@
 ---
-title: Send One-time Codes via Your Own SMS Gateway
+title: Set Up SMS Gateway for Passwordless Connections
 topics:
     - connections
     - passwordless
     - sms
 contentType: how-to
 useCase: customize-connections
 ---
-# Send One-Time Codes via Your Own SMS Gateway
+# Set Up SMS Gateway for Passwordless Connections
 
-By default, the SMS connection uses Twilio to send the one-time code. If you already have your own infrastructure to send text messages, you cannot or might not want to use Twilio.
+If you are using a Passwordless connection, this guide will show you how you can use custom infrastructure that you've built (instead of Twilio) to send out one-time use codes via SMS messages to users who are logging in.
 
-Instead, you'll want your own infrastructure to handle the delivery of one-time codes.
+1. Get an [Access Token for Management API](/api/management/v2/tokens). You will need this to make calls to the Management API to update your Passwordless connection.
 
-## Configure your SMS Gateway
+2. Obtain information about the Passwordless SMS connection.
 
-You will need to configure your own SMS Gateway through the Management API. 
+    To do this, make a GET call to the [Connections](/api/management/v2#!/Connections/get_connections) endpoint. The response from the endpoint will be an array of objects. Each object represents one connection affiliated with your tenant.
```

Comment:
Maybe add a description of how to make a call as in this guide:
https://auth0.com/docs/api/management/guides/roles/create-roles

I don't know. I'm on the fence about including these because our API Explorers should take care of this for people, but until our API docs are a little better, I've been told they are necessary. Thoughts?